mathlibtools/leanproject.spec

```diff
@@ -0,0 +1,39 @@
+# -*- mode: python ; coding: utf-8 -*-
+"""
+Originally generated with `pyinstaller`.
```

Comment:
@hmonroe, do you remember the exact command you used to generate this? If so, can you edit this line to describe it?

Reply:
Yes I added the command to generate the .spec file